registry/api/errcode/errors_test.go

```diff
@@ -163,3 +166,4 @@ func TestMarshalUnmarshal(t *testing.T) {
 		t.Fatalf("errors not equal after round trip: %#v != %#v", unmarshaled, errors)
 	}
 }
+*/
```

Comment:
`TestMarshalUnmarshal` can be refactored to remain in this package. We may want to actually leave a version of this in the `v2` package to ensure serialization is not changing.